### DISCUSSION TITLE
An error fp calcualtion at remote cluster using PBS

### DISCUSSION THREAD

**Discussion by Zhhaibing** [deepmodeling/dpdispatcher#264] [Q&A]
The dpgen excute the fp calculation at the queue ljcluster, on another cluster using PBS. when the dpgen program try to get the job state (Q、H or R)  it return an error:
![3e726b9f56a99d8007f562d7f1f7565](https://user-images.githubusercontent.com/40423083/193561006-197cfbf8-7e5c-4b70-be6c-af9fc5d9bb11.jpg)
I check the *.sub file according to the bug tips and find that:
![1664794718764](https://user-images.githubusercontent.com/40423083/193561652-2777f2a4-df4d-4cf5-a8bd-84ba96666bca.jpg)
that's different with PBS I use as usual:
![1664796439004](https://user-images.githubusercontent.com/40423083/193566088-62129094-d124-4292-8f97-40cb97d651fe.jpg)
In fact, a task have been submitted by using "qstat |grep ljcluster":
![image](https://user-images.githubusercontent.com/40423083/193562819-fe7f1ab9-d489-47e6-aef9-ffdec7c8e96d.png)
the job Wangzr_LJ is a normally running task on our cluster of other people and  the zhb_LJ is the fp task submitted by dpgen spaning cluster. 
I try to excute the command"qstat -x + job ID " used to get the my job state according to source code of the pbs.py:
![image](https://user-images.githubusercontent.com/40423083/193562211-2b0a5706-ae03-44e9-8ced-145a2be32479.png)
it return a line instead instead of multiple lines as the pbs.py expected.
For comparation, i use qstat -a |grep ljcluster to get the source the task expand and return:
![image](https://user-images.githubusercontent.com/40423083/193563778-4498325d-3381-4a54-a269-51a9a89ff23a.png)
the normally running task of wangzr return the source, but my task. And when i use the usual PBS script, the task runs normally, so do i need change the pbs format as that i use or some problems happened in my input files these are attached as follows. My dpgen version is 0.10.6.
[MACHINE.txt](https://github.com/deepmodeling/deepmd-kit/files/9697001/MACHINE.txt)
[PARAM.txt](https://github.com/deepmodeling/deepmd-kit/files/9697002/PARAM.txt)

**Comment by njzjz** [comment]
What is your PBS version?

**Comment by Zhhaibing** [comment]
and I found the dpgen task will break down at the stage when get the task state with "status_word=status_line.split()[-2]"

**Comment by njzjz** [comment]
Do you use torque? "PBS" is designed for openpbs or PBS Pro. For torque please use "Torque".
